.travis.yml

```
@@ -45,13 +45,19 @@ matrix:
     <<: *OPAM
   - env:
     - COQ_IMAGE=coqorg/coq:8.10
```

Same as other comment.
```suggestion
    - COQ_IMAGE=mathcomp/mathcomp:1.10.0-coq-8.10
```